rtl/eclrtl/rtlds.cpp

```diff
@@ -1892,3 +1892,14 @@ void MemoryBufferBuilder::finishRow(size32_t length)
     self = NULL;
     reserved = 0;
 }
+
+void MemoryBufferBuilder::removeBytes(size32_t len)
+{
+    dbgassertex(buffer);
+    if (reserved>len)
```

Comment:
What are these two lines here for?   I would have expected < rather than >

Reply:
I think it should be:
```
   assertex(len >= reserved);
```